spec/takeaway_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'takeaway.rb'
+require 'menu.rb'
+
+describe Takeaway do
+  let(:takeaway) { Takeaway.new }
+  let(:menu) { Menu.new }
+  let(:order) { Order.new }
+
+  it { is_expected.to respond_to :menu }
```

Comment:
These one line tests are great, I haven’t used them before myself but they’re very clear so I'll definitely have a go at this going forward.